generateQuestions.js

```diff
@@ -0,0 +1,15 @@
+const handleObject = require('./handleObject');
```

Comment:
Nice use of creating questions through iteration. Does this allow users to enter in more than one experience or education field if they want to though?

Reply:
Thank you! Yes that was the purpose of this architecture. I didn't get a chance to implement the handleArray function which would allow users to enter more than one entry.